apps/wallet/src/ui/app/pages/initialize/import/steps/StepOne.tsx

```diff
@@ -60,31 +58,31 @@ export default function StepOne({ next, data, mode }: StepProps) {
                                 );
                                 handleBlur(e);
                             }}
-                            className={st.mnemonic}
+                            className="text-steel-dark flex flex-col flex-nowrap gap-2 self-stretch font-semibold text-heading5 p-3.5 rounded-[15px] bg-white border border-solid border-[#EBECED] shadow-button leading-snug resize-none min-h-[100px] placeholder:text-steel-dark"
```

Comment:
Looks like `rounded-15` is already defined
Also checking the figma design `#EBECED` is gray-45. Maybe it changed or our tailwind config is wrong. Can we change here to border-gray-45 and fix tailwind config as well? 
<img width="218" alt="Screenshot 2023-01-18 at 13 12 19" src="https://user-images.githubusercontent.com/10210143/213182479-1588ea35-c41c-4ced-83a3-200dbb5a6301.png">